src/nodes/mono_camera_node.cpp

```diff
@@ -1,15 +1,26 @@
 #include <ros/ros.h>
 #include <avt_vimba_camera/mono_camera.h>
+#include <avt_vimba_camera/status_camera.h>
 
 int main(int argc, char** argv)
 {
-	ros::init(argc, argv, "mono_camera_node");
-
+  ros::init(argc, argv, "mono_camera_node");
   ros::NodeHandle nh;
   ros::NodeHandle nhp("~");
-
+  ros::Rate loop_rate(10);
```

Comment:
I think 10hz loop_rate might cap our frame rate to 10fps. We need to either use the free spin() function or modify to a higher spin rate.

Reply:
19